contrib/kind.sh

```diff
@@ -168,8 +166,6 @@ parse_args() {
                                                 ;;
             -ds | --disable-snat-multiple-gws ) OVN_DISABLE_SNAT_MULTIPLE_GWS=true
                                                 ;;
-            -dp | --disable-pkt-mtu-check )     OVN_DISABLE_PKT_MTU_CHECK=true
```

Comment:
should we keep this config option in case someone is still using it even with the new way of handling it still seems applicable? I cant remember why I added this in the first place.

Reply:
kept it